addons/boa/settings/defaults.py

```diff
@@ -0,0 +1,32 @@
+DEFAULT_HOSTS = []
```

Comment:
Yeah, it looks like we just use `boaapi.boa_client.BOA_API_ENDPOINT`, so these can be removed

Reply:
Yep, this was leftovers from the owncloud base.

Reply:
Slight changed of heart: instead of removing, added a comment to the settings & local-dist files in [579bd4d](https://github.com/CenterForOpenScience/osf.io/pull/10484/commits/579bd4dfbacd074fa260b3011475dc095d0784fe).  If we decide to add support for other Boa dbs in the future, this is where we'll need to add them. Leaving them in will help us keep track.